kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -1359,9 +1359,19 @@ def _kill_vm(self):
         for covering_path in self.covering_paths:
             running_vm_uuids.update(find_ps_running_vm(covering_path))
 
+        logger.info("cbd fencer found running vms on ps %s: %s" % (self.ps_uuid, running_vm_uuids))
         for vm_uuid in running_vm_uuids:
             pid = linux.get_vm_pid(vm_uuid)
-            linux.kill_process(pid)
+            try:
+                logger.info("cbd fencer try to kill vm %s with pid %s" % (vm_uuid, pid))
+                linux.kill_process(pid, is_graceful=False)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)